urbansim_parcels/utils.py

```diff
@@ -770,7 +781,7 @@ def run_feasibility(parcels, parcel_price_callback,
     pf = (sqftproforma.SqFtProForma.from_yaml(str_or_buffer=cfg)
           if cfg else sqftproforma.SqFtProForma.from_defaults())
     sites = (pl.remove_pipelined_sites(parcels) if pipeline
-             else parcels.to_frame())
+             else parcels.local)#to_frame())
```

Comment:
Typo?

Reply:
Solved in https://github.com/urbansim/urbansim_parcels/pull/32/commits/2a23d734298cb1650a80beb45d41acb88d8371d1